src/ui/NoNameFacet/NoNameFacetHeader.ts

```diff
@@ -88,23 +92,66 @@ export class NoNameFacetHeader {
   }
 
   private recreateOperator() {
-    $$(this.operatorElement).replaceWith(this.createOperator());
+    const newElement = this.createOperator();
+    $$(this.operatorElement).replaceWith(newElement);
+    this.operatorElement = newElement;
+  }
+
+  private createCollapse() {
+    const { enableTogglingCollapse, isCollapsed } = this.options.rootFacetOptions;
```

Comment:
Instead of `enableTogglingCollapse`, I prefer the option name in the documentation `enableCollapse`. Would it be possible to rename to that or was there a reason for using this name?